src/pg_timestampz.erl

```diff
@@ -12,11 +13,14 @@
 init(_Opts) ->
     {[<<"timestamptz_send">>], []}.
 
+
+encode({{Year, Month, Day}, {Hour, Minute, Second}, UtcOffset}, TypeInfo) ->
+    encode({{Year, Month, Day}, {Hour + UtcOffset, Minute, Second}}, TypeInfo);
 encode(Timestamp, _TypeInfo) ->
     <<8:?int32, (pg_timestamp:encode_timestamp(Timestamp)):?int64>>.
 
 decode(Bin, _TypeInfo) ->
     pg_timestamp:decode_timestamp(Bin, []).
 
 type_spec() ->
-    pg_timestamp:type_spec().
+    "{{Year::integer(), Month::1..12, Day::1..31}, {Hours::integer(), Minutes::integer(), Seconds::integer() | float()}, UtcOffset::integer()}".
```

Comment:
I don't really understand what this defines. It is possible it should be the `pg_timestamp::type_spect()` one **plus** the version with the UtcOffset.